https://issues.redhat.com/browse/ACM-17241 troubleshooting import cus… (#7542)
open-cluster-management/rhacm-docs: clusters/support_troubleshooting/trouble_cluster_cert_update.adoc

> [#troubleshooting-ca-update-on-hive-and-assisted-installer-clusters]

Reviewer: these ID's should be short, I will changed them.
```suggestion
[#ca-update-hive-assisted-installer-clusters]
```

= Troubleshooting certificate authority update on Hive and Assisted Installer clusters

> 

Reviewer: We need a description in between headers always. See the standard here:

https://www.ibm.com/docs/en/ibm-style?topic=format-headings#dividing-information

"Do not place one heading directly after another. Ensure that each heading is followed by text, or consider whether one heading can be removed. If the text that separates the headings is trivial, one of the headings is likely unnecessary."

> = Troubleshooting CA update on Hive and Assisted Installer clusters

Reviewer: This should be spelled out, assuming this is cert. auth.
```suggestion
= Troubleshooting certificate authority update on Hive and Assisted Installer clusters
```

== Symptom: Connection error message appears after you update the cluster CA on the hive-provisioned or assisted installer cluster

When you update the cluster CA with your enterprise CA on the hive-provisioned or assisted installer cluster, the corresponding cluster deployment on the hub displays a connection error message:
----
x509: certificate signed by unknown authority
----
If you back up and restore the cluster in this state, you cannot automatically import the managed cluster to the restored hub.

== Resolving the problem: Connection error message appears after you update the cluster CA on the hive-provisioned or assisted installer cluster

> . Define your custom CA with the global variable in your system:

Reviewer: We need to add the action: Run the following command: 

I will handle that change in a commit.

+

> [literal,subs="+quotes,attributes",options="nowrap",role=white-space-pre]

Reviewer: We can remove all of line 16.

----

> export CA=<__your_customized_CA__>

Reviewer: ```suggestion
export CA=<your_customized_CA>
```

----

> . Generate a new secret with your customized CA:

Reviewer: Run the following command. (A nonsighted user has no idea at this line what they do to generate, that is why we make the step clear they are about to run a command,

+
[source,terminal]
----

oc create secret generic additional-ca \
  --from-literal=ca.crt="$CA" \
  --namespace hive

----

. Update your cluster with the new secret:
+
[literal,subs="+quotes,attributes",options="nowrap",role=white-space-pre]
----

oc patch hiveconfig hive --type=merge -p '
{
  "spec": {
    "additionalCertificateAuthoritiesSecretRef": [
      {
        "name": "additional-ca"
      }
    ]
  }
}'

> ----

Reviewer: We are missing some header IDs. I will handle that as much as I can with the draft.
